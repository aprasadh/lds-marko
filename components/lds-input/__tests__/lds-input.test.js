const component = require('lds-input/index.marko');
describe('component', () => {
    it('should have a renderToString function', () => {
      expect(component.renderToString()).toBeDefined();
    });

    it('should render correctly', () => {
        component.renderToString({
          label: 'First Name',
          placeholder: 'first name'
        }, (err, html) => {
          expect(html).toMatchSnapshot();
        });
      });
});
