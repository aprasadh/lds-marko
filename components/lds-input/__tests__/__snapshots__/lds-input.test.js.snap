// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component should render correctly 1`] = `"<!--M#s0--><div class=\\"slds-form-element\\"><label class=\\"slds-form-element__label\\" for=\\"s0-input-text\\">First Name</label><div class=\\"slds-form-element__control\\"><input type=\\"text\\" class=\\"slds-input\\" placeholder=\\"first name\\" value=\\"\\" id=\\"s0-input-text\\"></div></div><!--M/s0-->"`;
